index.html

```diff
@@ -214,8 +213,8 @@ <h1 class="text-white mb-4">Features</h1>
           <div class="bg-black text-center h-100 project">
             <div class="d-flex h-100">
               <div class="project-text w-100 my-auto text-center text-lg-right">
-                <p class="text-white">Coolness</p>
-                <p class="mb-0 text-white-50">There is a new rover in town!</p>
+                <p class="text-white">Fun</p>
+                <p class="mb-0 text-white-50">There is a new rover in town! ExoMy can be steered with a gamepad or a website for smartphones, tables and laptops.</p>
```

Comment:
Always wanted to steer ExoMy with a table :)

Reply:
```suggestion
                <p class="mb-0 text-white-50">There is a new rover in town! ExoMy can be steered with a gamepad or a website for smartphones, tablets and laptops.</p>
```